pkgs/tools/misc/nncp/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, go, fetchurl, curl, perl, genericUpdater, writeShellScript }:
+
+stdenv.mkDerivation rec {
+  pname = "nncp";
+  version = "5.3.3";
+
+  buildInputs = [ go ];
```

Comment:
This should be `nativeBuildInputs`.

Also, could you please reorder the attributes as in https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887 please?

Reply:
done. thanks for being reviewed so hard. :-)